boards/arm/mimx8mq_evk/Kconfig.board

```diff
@@ -0,0 +1,9 @@
+# MIMX8MQ EVK board
+
+# Copyright (c) 2021, Kwon Tae-young <tykwon@m2i.co.kr>
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_MIMX8MQ_EVK
```

Comment:
It's possible we may want to support the A53 cores in the future, so could you please rename the board configuration with a `_cm4` suffix? Keep the board folder name the same (`boards/arm/mimx8mq_evk`), but rename `mimx8mq_evk_defconfig` to `mimx8mq_evk_cm4_defconfig`; this will leave room for `mimx8mq_evk_a53_defconfig` to be added later.

Some examples of other boards with multiple core configurations:
  - `mimxrt1170_evk`
  - `lpcxpresso54114`
  - `lpcxpresso55s69`

Reply:
Done